<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>globedialog</class>
 <widget class="QDialog" name="globedialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>球体积计算</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>141</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/iocFile/images/4.ico);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableWidget" name="globeTbWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>531</width>
     <height>192</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>编号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>名称</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>半径</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>体积</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>140</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="lookButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>140</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>121</x>
     <y>85</y>
     <width>75</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>球的体积：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>121</x>
     <y>29</y>
     <width>75</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>球的半径：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="radiusEdit">
   <property name="geometry">
    <rect>
     <x>221</x>
     <y>29</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="calculateButton">
   <property name="geometry">
    <rect>
     <x>121</x>
     <y>141</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>计算</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton_2">
   <property name="geometry">
    <rect>
     <x>221</x>
     <y>141</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="volumeEdit">
   <property name="geometry">
    <rect>
     <x>221</x>
     <y>85</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
